packager/media/codecs/aac_audio_specific_config.cc

```diff
@@ -205,6 +206,18 @@ uint8_t AACAudioSpecificConfig::GetNumChannels() const {
   return num_channels_;
 }
 
+bool AACAudioSpecificConfig::ParseAudioObjectType(BitReader* bit_reader) {
+  RCHECK(bit_reader->ReadBits(5, &audio_object_type_));
+  
+  if (audio_object_type_ == 31) {
```

Comment:
Should we maybe use the constant name here?  (Is this AOT_ESCAPE?)

Reply:
there are other integer references in the file (lines 286, 290, 295, 296, for example). While I'm in there, would you like me to switch those to constants?